src/main/java/org/sputnikdev/bluetooth/gattparser/spec/BluetoothGattSpecificationReader.java

```diff
@@ -221,6 +226,54 @@ public void loadExtensionsFromFolder(String path) {
         readCharacteristics(getFilesFromFolder(characteristicsFolderName));
     }
 
+    private static URL specResourceURL(URL original, String characteristicType) throws MalformedURLException {
+        String oldfile = original.getFile();
```

Comment:
Same thing as above, I'd prefer to use camel case names, e.g. `oldFile`
There are some other instances below as well

Reply:
ok, renamed the locals to better reflect what they hold